compiler/ETA/CodeGen/Utils.hs

```diff
@@ -24,7 +25,9 @@ cgLit (MachWord64 i)        = (jlong, lconst $ fromIntegral i)
 cgLit (MachFloat r)         = (jfloat, fconst $ fromRational r)
 cgLit (MachDouble r)        = (jdouble, dconst $ fromRational r)
 -- TODO: Remove this literal variant?
-cgLit MachNullAddr          = (jobject, aconst_null jobject)
+cgLit MachNullAddr          = (jobject, nullAddr)
+  where nullAddr = getstatic $ mkFieldRef "eta/runtime/io/MemoryManager"
+                   "nullAddress" (obj "java/nio/ByteBuffer")
```

Comment:
Couple changes:
- `(obj "java/nio/ByteBuffer")` -> `byteBufferType` (defined in `ETA.CodeGen.Rts`)
- `"eta/runtime/io/MemoryManager"` -> `memoryManager` (defined in `ETA.CodeGen.Rts`)